update_data.py

```diff
@@ -7,8 +7,9 @@
 #pylint: disable=wrong-import-position,import-error
 sys.path.append('/afs/cern.ch/cms/PPD/PdmV/tools/McM/')
 from rest import McM
+#XSDB from update twiki
+from update_twiki import get_xs
```

Comment:
Please move this outside the `#pylint: disable=...` and `#pylist: enable=...`. These two comments disable pylinter for these two special lines where path variable is edited and McM is imported from afs.